challenges/branches_arent_just_for_birds.html

```diff
@@ -192,6 +192,9 @@
 
   <p>To fix a remote that is pointing to the wrong place you can re-set its url: <code>git remote set-url origin <URLTOYOURFORK></code>.
 
+  <h4>Authentication failed...error: 401</h4>
+  <p>Your identity could not be verified. You may have 2FA turned on in which case you must use a personal access token or push through the GitHub for Mac or GitHub for Windows which handles 2FA for you.</p>  
```

Comment:
I'm wondering how to approach this topic without making things too complex. I'd like to nix the part about the desktop apps since Git-it isn't teaching people how to use those, so I don't want to send them into a new world unprepared. 

It may also be reasonable to assume people won't know how to get a personal access token so I think we should explain or link to help for doing that. 

Here's a proposed revision:

> Your identity could not be verified. You may have 2FA (Two Factor Authentication) turned on in which case you must use a personal access token as your password. You can generate one by [following these instructions](https://help.github.com/articles/creating-an-access-token-for-command-line-use). You'll need to keep this and use it in place of your password when prompted. You can also [save this access token](https://help.github.com/articles/caching-your-github-password-in-git/) on your computer so that you don't have to re-enter it.